bdk-ffi/src/tx_builder.rs

```diff
@@ -232,6 +234,13 @@ impl TxBuilder {
         })
     }
 
+    pub(crate) fn version(&self, version: i32) -> Arc<Self> {
+        Arc::new(TxBuilder {
+            version: Some(version),
+            ..self.clone()
+        })
+    }
+
```

Comment:
Was able to write two tests (one for version 1 and one for version 2) and both passed, looks good!